### PR TITLE
Update SolarWinds Host Agent to 3.1.1.717 version  (AO-16530)

### PR DESCRIPTION
**Pull request by d-maslyk**
* Update SolarWinds Host Agent to 3.1.1.717 version

* Update README 

* Whitespace changes for yaml files - most missing indentation related

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,14 +285,15 @@ Version 22 of Kubernetes collector allows you to collect cluster events and push
 
   kubectl describe configmaps -n kube-system plugin-configs task-manifests
   ```
-* Edit `configmaps/appoptics-token.yaml` and insert your `APPOPTICS_TOKEN`.
-* Create ServiceAccount and Deployment:
+* Create Kubernetes secret for APPOPTICS_TOKEN
```
Comment (lootek) [inline README.md:288]:
```suggestion
* Create Kubernetes secret for SOLARWINDS_TOKEN
```

?

Comment (lootek) [reply README.md:288]: hm, to be discussed with @tomekkolo and @melord as there are many more occurrences

Comment (tomekkolo) [reply README.md:288]: yes, SOLARWINDS_TOKEN should be used everywhere

File: conf/swisnap-init.sh [hunk 2]
```diff
@@ -2,107 +2,111 @@
...
     fi
 
     if [ "${SWISNAP_ENABLE_MESOS}" = "true" ]; then
-        mv ${PLUGINS_DIR}/mesos.yaml.example ${PLUGINS_DIR}/mesos.yaml
+        mv "${PLUGINS_DIR}/mesos.yaml.example" "${PLUGINS_DIR}/mesos.yaml"
     fi
 
     if [ "${SWISNAP_ENABLE_MONGODB}" = "true" ]; then
-        mv ${PLUGINS_DIR}/mongodb.yaml.example ${PLUGINS_DIR}/mongodb.yaml
+        mv "${PLUGINS_DIR}/mongodb.yaml.example" "${PLUGINS_DIR}/mongodb.yaml"
```
Comment (tomekkolo) [inline conf/swisnap-init.sh:67]: would be good to add NGINX and NGINX PLUS

File: README.md (hunk 1 above)
Comment (tomekkolo) [reply README.md:288]: generally, SOLARWINDS_TOKEN is the variable to be used everywhere, but for backward compatibility APPOPTICS_TOKEN should also be accepted inside container. Initi script should require either variable to be present https://github.com/solarwinds/solarwinds-snap-agent-docker/blob/1ec91aceb000de94f3b63eeb32be9c4ea8d58321/conf/swisnap-init.sh#L10

File: conf/swisnap-init.sh (hunk 2 above)
Comment (d-maslyk) [reply conf/swisnap-init.sh:71]: Request moved to backlog.

File: README.md (hunk 1 above)
Comment (d-maslyk) [reply README.md]: Added handling SOLARWINDS_TOKEN and moved APPOPTICS_TOKEN to be used as an fallback option.

File: README.md [hunk 3]
```diff
@@ -16,16 +30,15 @@ Alternatively, you can deploy the containerized agent in a sidecar to run the ot
 
 ## Installation
 
-All deployments expect an appoptics-token secret to exist.  You can create this secret via
+All deployments expect an appoptics-token secret to exist. You can create this secret via
```
[the remark this answers is not in the document]
Comment (d-maslyk) [reply README.md]: Changed.

**Pull request merged**